### DISCUSSION TITLE
Random series loading in view port

### DISCUSSION THREAD

**Discussion by Ravindra-Desai128** [nroduit/Weasis#545] [Q&A]
Hi @nroduit, 
when loading series from the PACS, it's randomly appearing in the viewport with a yellow indication. I need the first series to always load in view. I'm unable to figure out the logic behind this. Could you please help me?

**Comment by nroduit** [comment]
As series downloads are currently asynchronous and parallel processes, the first series downloaded is displayed. The result may therefore differ from one download to the next.

As already mentioned, the management of these aspects could be done in the hanging protocols for which I can't give you a realization date.

To display the first series, you can also use the page-up [shortcut](https://weasis.org/en/basics/shortcuts/index.html#selected-view-of-the-2d-dicom-viewer).
